src/toil/common.py

```diff
@@ -796,13 +797,19 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         """
         Clean up after a workflow invocation. Depending on the configuration, delete the job store.
         """
+        from toil.utils.toilStatus import ToilStatus
```

Comment:
I don't think that this is being used anymore, so it should probably be removed.